libreoffice-core/writerfilter/source/ooxml/OOXMLDocumentImpl.cxx

```diff
@@ -516,14 +516,19 @@ void OOXMLDocumentImpl::resolve(Stream & rStream)
     }
     catch (xml::sax::SAXException const& rErr)
     {
+
+        // MACRO: silently swallow there - we don't have access to prompt the user
+        // better to try and load it then to fail silently {
+        SAL_WARN("writerfilter.ooxml", "Unhandled SAXException: " << rErr.Message);
```

Comment:
should we be worried about the warning below that says to not silently swallow these?